app/src/main/res/values-es/help.xml

```diff
@@ -257,51 +257,51 @@
-        by asking those for their peers, and so on. If you have no bookmarked nodes
-        (or they don&apos;t tell us about their peers),
-        Monerujo will go straight to the Monero seed nodes hardcoded into Monero. The
-        scan stops when it finds 10 remote nodes in total.</p>
+        <p>Refresca la lista de nodos al mostrar los marcadores de nodos para permitir a Monerujo
+        elegir el mejor para ti.</p>
+        <h2>¿Qué es un Nodo?h2>
+        <p>Monerujo usa un Nodo Remoto (algunas veces también llamado Daemon) para comunicarse con
+        la red de Monero sin tener que descargar y almacenar una copia de la
+        cadena de bloques completa.<p>
+        <h2>Lista de Nodos</h2>
+        <p>Si la lista está vacía, puedes agregar nuevos nodos manualmente o dejar que Monerujo
+        escanee la red por ti. O ambos.</p>
+        <p>La lista de nodos muestra todos los nodos actualmente conocidos. Adicionalmente, la estampa de tiempo
+        del último bloque conocido para cada nodo se muestra bajo el nombre del nodo. Un icono
+        representando el comportamiento de respuesta del nodo
+        (el cual indica el nivel de conectividad esperado)
+        es mostrado seguido de cada nodo.</p>
+        <p>Cualquier nodo en la lista puede ser marcado para un posterior uso.
+        Nodos que no sean marcados serán olvidados.<p>
+        <p>Monerujo elegirá el nodo marcado óptimo cada vez que lo uses.
+        Esto lo hace revisando la altura del bloque (¿qué tan al día
+        está el nodo?) así como el comportamiento de respuesta (¿qué tan rápido responde el nodo a solicitudes?).</p>
+        <p>La lista se ordena por estas características, por lo tanto el primer nodo en la lista sería el que Monerujo
+        elegiría en ese momento. El final de la lista mostraría nodos muy lentos o no disponibles.</p>
+        <h2>Agregar un Nodo</h2>
+        <p>Presionando en el botón &quot;Agregar Nodo&quot; en la parte inferior, se te pedirá que
+        ingreses los detalles del nodo en el siguiente cuadro de diálogo.
+        La &quot;Dirección&quot; es el nombre del host o dirección IP del nodo - esta es la única
+        entrada obligatoria.
+        Ingresa el &quot;Puerto&quot; si el nodo se ejecuta en un puerto que no es por defecto (e.g. 18089).
+        Puedes también nombrar el nodo opcionalmente, para que puedas identificarlo después fácilmente.
+        Algunos nodos requieren credenciales para usarlos. Ingresa el nombre de usuario provisto &amp;
+        contraseña en los campos apropiados. Ahora puedes &quot;Verificar&quot; estos ajustes.
+        Los &quot;Resultados de la Prueba&quot; mostrarán la altura del bloque, tiempo de respuesta y dirección IP real usados.
+        El resultado puede ser también un error - usualmente porque el nombre de host provisto no
+        es alcanzable en una sensible cantidad de tiempo o las credenciales son incorrectas.
```

Comment:
Suggestion:

"...en una **cantidad de tiempo sensible**..."

Reply:
Will leave it as is, an adjective can be used before or after the subject, sometimes.